site/themes/arangodb-docs-theme/static/css/theme.css

```diff
@@ -144,6 +144,44 @@ button, input, optgroup, select, textarea {
     margin-bottom: 15px;
 }
 
+.button {
+    background: transparent;
+}
+
+.button__wrapper {
+    border-radius: 4px;
+    text-align: center;
+    text-decoration: none;
+    border: none;
+    cursor: pointer;
+    width: fit-content;
+    height: 2rem;
+}
+
+.button__wrapper__green {
+    background-color: var(--HEADERS-COLOR);
+}
+
+.button__wrapper__grey {
+    background-color: var(--gray-200);
+}
+
+.button__link {
+    padding: 5px 1em 5px 1em;
+    vertical-align: middle;
+}
+
+.button__text {
+    border-bottom: 2px solid var(--green-600);
```

Comment:
Can this not be a part of `button` itself?

Reply:
Changing it to be part of button will display the border bottom with size of the entire button. By keeping as it is the border will be the size of the text